src/enemy/gyorgMale.c

```diff
@@ -26,7 +26,6 @@ const u8 gUnk_080D1C44[0xC];
 const u16 gUnk_080D1C50[8];
 const u16 gUnk_080D1C60[8];
 
-void GyorgMale(Entity*);
```

Comment:
Same thing as in Manager15, I think these forward declarations could stay.

Reply:
this also has a public declaration in enemy.h (with types this time)
so declaring it here would be entirely redundant